index.html

```diff
@@ -5,8 +5,124 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <header class="pege-header">
+      <nav class="navigation">
+          <img src="logo.svg" alt="Город Седона" width="40" height="30">
+        <ul class="navigation-list">
+           <li class="navigaton-item">
+             <a class="navigation-link" href="#">Главная</a>
+             </li>
+           <li class="navigaton-item">
+             <a class="navigation-link" href="#">О Седоне</a>
+             </li>
+           <li class="navigation-item">
+             <a class="navigation-link" href="catalog.html">Гостиницы</a>
+             </li>
+        </ul>
+        <ul class="navigation-user">
+          <li class="navigaton-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Поиск</span>
```

Comment:
```suggestion
              <span class="visually-hidden">Поиск.</span>
```

во всех вижуали хидден тоже ставим точки, чтобы скрин-ридер делала паузу

Reply:
нужно поправить везде